src/main/java/com/icthh/xm/ms/timeline/config/lep/LepContext.java

```diff
@@ -4,10 +4,9 @@
 import com.icthh.xm.commons.config.client.api.ConfigService;
 import com.icthh.xm.commons.config.client.service.TenantConfigService;
 import com.icthh.xm.commons.lep.api.BaseLepContext;
-import com.icthh.xm.commons.logging.trace.TraceService.TraceServiceField;
 import com.icthh.xm.ms.timeline.service.cache.EventDeduplicationStrategyFactory;
 
-public class LepContext extends BaseLepContext implements TraceServiceField {
+public class LepContext extends BaseLepContext {
```

Comment:
Why TraceServiceField removed? O_o

Reply:
because TraceServiceField is not present in xm-commons version 4.x

Reply:
this service present in  commons master